content/docs/user-guide/assets/pipeline/asset-builders.md

```diff
@@ -1,71 +1,190 @@
+| project-cache-path | Full path to the project cache folder.                                                                                                                                                                        |
+| module             | For resident mode, the path to the builder dll folder, otherwise the full path to a single builder dll to use.                                                                                                |
+| port               | Optional, port number to use to connect to the Asset Processor.                                                                                                                                               |
+| id                 | UUID string that identifies the builder.  Only used for resident mode when the Asset Processor directly starts up the AssetBuilder.                                                                           |
+| input              | For non-resident mode, full path to the file containing the serialized job request.                                                                                                                           |
+| output             | For non-resident mode, full path to the file to write the job response to.                                                                                                                                    |
+| debug              | Debug mode for the create and process job of the specified file. <br/>Debug mode optionally uses `-input`, `-output`, `-module`, `-port` and `-gameroot`. For example: `-debug Objects\Tutorials\shapes.fbx`. |                                                                                                                                  
+| debug_create       | Debug mode for the create job of the specified file.                                                                                                                                                          |
+| debug_process      | Debug mode for the process job of the specified file.                                                                                                                                                         |
+| tags               | Additional tags to add to the debug platform for job processing. One tag can be supplied per option.                                                                                                          |
+| platform           | Platform to use for debugging, for example, `pc` |
+
+## Loading other files
+
+Sometimes, when authoring a builder, it may be necessary to load other files besides the primary source file to finish a given process. Here are some scenarios that this can come up, and best practices for handling them.

```

Comment:
The second sentence seems out of place with the new organization.

```suggestion
Sometimes, when authoring a builder, it may be necessary to load other files besides the primary source file to finish a given process.
```